CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ We use the following categories for changes:
 - Log warning when failing to write response to remote read requests [#1180]
 - Fix Promscale running even when some component may fail to start [#1217]
 - Fix `promscale_ingest_max_sent_timestamp_milliseconds` metric for tracing [#1270]
+- Fix `prom_api.reset_metric_retention_period` on two-step continuous aggregates [#1294]
```

Comment:
How is this a FIX? It's just a test change

Reply:
https://github.com/timescale/promscale_extension/pull/180

Reply:
Ok. I've found a related extension change here https://github.com/timescale/promscale_extension/pull/180/files
But still let's add some description explaining the problem and the fix.  From extension PR it seems there was some problem with migration script (a wrong column name) if I am not mistaken

Reply:
done

Reply:
btw I am thinking if we should track extension changes in their own CHANGELOG? Obviously the fix in the extension can be released separately from Promscale connector so I am not sure why are we coupling these two? I am assuming that being able to release extension separately from connector is better (obviously in some cases that will not be possible...). Thoughts?